snapcraft.yaml

```diff
@@ -66,6 +66,11 @@ apps:
     restart-condition: always
     plugs: [network, network-bind]
 
+  nextcloud-cron:
+    command: nextcloud-cron
+    daemon: simple
+    restart-condition: always
```

Comment:
This actually requires the network, so please include `plugs: [network, network-bind]` here. Otherwise it's killed by seccomp.